dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -207,7 +207,7 @@ static inline std::tuple<ReadFromStreamResult, PrehandleResult> executeTransform
             res = ReadFromStreamResult{.error = abort_reason.value(), .extra_msg = "", .region = new_region};
         }
         auto keys_per_second = (sst_stream->getProcessKeys().write_cf + sst_stream->getProcessKeys().lock_cf
-                                + sst_stream->getProcessKeys().write_cf)
+                                + sst_stream->getProcessKeys().default_cf)
```

Comment:
😂 seems we should backport this to the affected release-x branch

Reply:
It's merged into master last week.

Reply:
> It's merged into master last week.

ref https://github.com/pingcap/tiflash/pull/9131